site/src/pages/docs/server-flags-provider.mdx

```diff
@@ -0,0 +1,78 @@
+# Configuring global flags
+
+## What flags in Armeria are
+
+You can configure the system properties by implementing your own <type://FlagsProvider>. It is useful for following two situations:
+
+1. To override the default system properties.
```

Comment:
`system properties` has a special meaning in Java set on the Java command line using the `-Dpropertyname=value` syntax. It might be better to use a different term to avoid confusion.
```suggestion
1. To override the default system values.
```